libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPrompt.cs

```diff
@@ -426,12 +433,103 @@ private async Task<PromptRecognizerResult<TokenResponse>> RecognizeTokenAsync(IT
                     }
                     else
                     {
-                        await turnContext.SendActivityAsync(new Activity { Type = ActivityTypesEx.InvokeResponse, Value = new InvokeResponse { Status = 404 } }, cancellationToken).ConfigureAwait(false);
+                        await this.SendInvokeResponseAsync(turnContext, cancellationToken, 404).ConfigureAwait(false);

```

Comment:
Use constant, HttpStatusCodes.NotFound already defined in the framework IIRC (name may vary slightly)